doc/manual/src/expressions/language-values.md

```diff
@@ -172,7 +180,7 @@ function and the fifth being a set.
 
 Note that lists are only lazy in values, and they are strict in length.
 
-## Attribute Sets
+## Attribute Set
 
 Attribute sets are collections of name-value-pairs (called *attributes*) enclosed in curly brackets (`{ }`).
```

Comment:
If you're de-pluralizing the heading, maybe the body should reflect that as well? (There may be more examples out of view of the diff.)

e.g.:
```suggestion
## Attribute Set

An attribute set is a collection of name-value-pairs (called *attributes*) enclosed in curly brackets (`{ }`).
```